src/link/link.cpp

```diff
@@ -37,7 +37,7 @@ Link::Link(LinkType linkType, QString name, QObject* parent)
         _abstractLink.reset(new Ping360SimulationLink());
         break;
     default :
-        qCDebug(PING_PROTOCOL_LINK) << "Link not available!";
+        assert("Link not available!");
```

Comment:
We should not assert in something that the user can fix, like choosing a different link or something. `assert` should only be used in places where it's not possible to recover.